package.json

```diff
@@ -108,7 +108,7 @@
     "prettier": "2.0.5",
     "pretty-bytes": "5.3.0",
     "pretty-ms": "7.0.0",
-    "react": "16.12.0",
+    "react": "17.0.1",
     "react-dom": "16.12.0",
```

Comment:
react-dom needs to be updated as well. But this needs more work given we already run tests on React 17. This is only for the tests.